fineract-provider/src/main/java/org/apache/fineract/adhocquery/domain/AdHoc.java

```diff
@@ -60,6 +60,10 @@
     @Column(name = "IsActive", nullable = false)
     private boolean isActive = false;
 
+    public AdHoc() {
+        super();
```

Comment:
if EclipseLink requires these, what I would do if I were you, to avoid conflicts with other PRs such as the Checkstyle work, is to raise a small PR focused on "add missing default constructors [FINERACT-849]" just to already get this done and in; it will likely be easier for you, and make future PRs smaller and easier for us to review.

Reply:
Ok. I saw an issue on that on jira. I will pick up on that.